examples/first-demo/controllers/posts.ts

```diff
@@ -23,7 +24,7 @@ export const PostsController = Controller(({db}) => ({
 
   async create(params, newData) {
     const data = await db.post.create({
-      data: newData,
+      data: newData as PostCreateInput,
```

Comment:
Instead of this change, we need to set up the types so that `newData` here automatically includes the attributes defined in `permit`